src/test/mockdata/articledata.js

```diff
@@ -0,0 +1,62 @@
+import slug from 'slug';
+import crypto from 'crypto';
+
```

Comment:
I think the methods here: https://github.com/andela/ah-kifaru-backend/blob/develop/src/test/utils/helpers.js already carter for a lot of the new objects introduced here

Reply:
Thank you for this feedback.
The methods you suggested is relatively inefficient to test some of the required behavior of the application using a TDD approach